db/seeds.rb

```diff
@@ -1,7 +1,49 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require "open-uri"
+
+10.times do |i|
+    print "Seeding." +  ("." * (i % 3)) + "  \r"
+    sleep(0.2)
+end
+
+puts 'Cleaning up database'
+Boat.destroy_all
+puts 'Database is cleaned'
+
+10.times do |i|
+    print "Creating some new nice Boats." +  ("." * (i % 3)) + "  \r"
+    sleep(0.2)
+end
+
+files = [
+  'https://www.bavariayachts.com/fileadmin/_processed_/f/8/csm_bavaria-sy-cline-c42-interior-02_bd508e7f7f.jpg',
+  'https://www.bavariayachts.com/fileadmin/_processed_/1/1/csm_bavaria-sy-cline-c42-exterior-image-01_686d32e4a7.jpg',
+  'https://www.wolf-angelsport.de/out/pictures/master/product/1/fox-fx--240-inflatable-boats-wolf-angelsport.jpg',
+  'https://www.berlin-bootsverleih.com/photos/boats/geilezeit-partytours/primary_thumb.jpg',
+  'https://q-cf.bstatic.com/images/hotel/max1024x768/178/178766063.jpg',
+  'https://www.seehaus.immobilien/content/index/62cf0145ce59a7271cb96e0d472dbc966170bc89.jpeg',
+  'https://i.pinimg.com/originals/15/0b/ff/150bffc448183c9a3bf2c7120af70518.jpg',
+  'https://i.pinimg.com/originals/c7/5a/af/c75aaf0ed67f697d3b9e260f901c8958.jpg',
+  'https://r-cf.bstatic.com/images/hotel/max500/104/104447609.jpg',
+  'https://excellenceriviera.com/wp-content/uploads/2020/05/Fjord-40-Featured.jpg',
+  'https://q-cf.bstatic.com/images/hotel/max1024x768/664/66467380.jpg',
+  'https://3brefc126v633i9r4zk9q3p5-wpengine.netdna-ssl.com/wp-content/uploads/2019/07/MGR00244-1920x1280.jpg',
+  'https://media.spacebase.com/media/cache/spaces/2442/tfo_seminarboot_wellenlaenge_2_1280.jpg',
+  'https://media-cdn.tripadvisor.com/media/photo-s/12/53/31/40/enjoy-a-picnic-while.jpg',
+  'https://media.tacdn.com/media/attractions-splice-spp-674x446/07/3b/0e/20.jpg',
+]
+
+20.times do
+  boat = Boat.create(
+    name: Faker::Name.female_first_name,
+    location: ['Berlin', 'Hamburg', 'Oslo', 'Amsterdam', 'Copenhagen'].sample,
+    capacity: rand(2..6),
```

Comment:
right now there are just 3 capacities in the boat model/validations and form to create a boat - 2/4/6. Can we change the rand to pick from those? Or we can change the validations and forms, but I think it makes more sense to change the seeding, since anyways maybe people go in couples.

Reply:
Yes I can create an array with [2, 3 , 6] and take one integer of those as capacity randomly.